solidity/ecdsa/contracts/WalletRegistry.sol

```diff
@@ -91,6 +115,13 @@ contract WalletRegistry is Ownable {
         Wallets.Signature signature
     );
 
+    event DkgMaliciousResultProved(
```

Comment:
I think `DkgMaliciousResultProved` fit a bit better, because it's more generic and we also have [this](https://github.com/threshold-network/solidity-contracts/blob/main/contracts/staking/TokenStaking.sol#L1541) event strickly telling about slashing. However, I agree that it might be confusing to have different event names between random-beacon and ecdsa